build/xwalk.gni

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This file contains Crosswalk-feature-related build flags. Users should
+# import it into their build folder by run:
+#   echo 'import("//xwalk/build/xwalk.gni")' > out/your_folder/args.gn'
+# Then run:
+#   gn gen out/your_folder
+
+# Set it false when building content_shell.
+declare_args() {
+  is_build_crosswalk = true
+}
+
+if (is_build_crosswalk) {
+  target_os = "linux"
+  target_cpu = "x64"
+  is_debug = false
+
+  # From src/build/config/features.gni
+  # TODO: Enable nacl for Linux.
+  enable_nacl = false
+
+  # From src/build/config/features.gni
+  enable_plugins = true
+
+  # From src/build/config/features.gni
+  enable_webvr = false
+
+  # From src/third_party/widevine/cdm/widevine.gni
+  enable_widevine = false
+
+  # From src/build/config/compiler/BUILD.gn
+  # If we want stack unwind support for backtrace()
+  exclude_unwind_tables = false
+
+  # From src/third_party/ffmpeg/ffmpeg_generated.gni
+  ffmpeg_branding = "Chromium"
```

Comment:
Sorry, there is some android flags.  My initial purpose to split out PR for Linux is to make it easier to review. So I didn't delete those android flags clean.  Anyway, you are right, it should have Linux related only in this PR.  I'll make it clean.